src/Pdf.php

```diff
@@ -12,38 +13,64 @@ class Pdf
 
     protected $binPath;
 
+    protected $options = [];
+
     public function __construct(string $binPath = null)
     {
         $this->binPath = $binPath ?? '/usr/bin/pdftotext';
     }
 
-    public function setPdf(string $pdf) : Pdf
+    public function setPdf(string $pdf) : self
     {
-        if (!file_exists($pdf)) {
-            throw new PdfNotFound("could not find pdf {$pdf}");
+        if (!\is_readable($pdf)) {
+            throw new PdfNotFound(\sprintf('could not find or read pdf `%s`', $pdf));
```

Comment:
I preferred the old way of formatted the exception message, could you revert this change?